src/shared_memory_manager.cc

```diff
@@ -60,72 +238,202 @@ SharedMemoryManager::GetCUDAHandle(
 {
   return TRITONSERVER_ErrorNew(
       TRITONSERVER_ERROR_UNSUPPORTED,
-      std::string("Shared memory feature is currently not supported on Windows")
+      std::string(
+          "GPU shared memory features are currently not supported on Windows")
           .c_str());
 }
+
 #endif  // TRITON_ENABLE_GPU
 
 TRITONSERVER_Error*
 SharedMemoryManager::GetMemoryInfo(
```

Comment:
Does this need to be implemented twice? I understand the iteration cycle you went though, that's fine to make sure it works, but there is no Windows specific code in this function.